packages/browser/src/plugins/in-app-plugin/index.ts

```diff
@@ -65,6 +65,17 @@ export function InAppPlugin(
                     'contentType': ContentType,
                 });
             }
+            if (settings.events) {
+                _eventTarget.dispatchEvent(newEvent(InAppEvents.MessageOpened, { 
+                    messageId: message?.messageId,
+                    deliveryId: message?.properties?.gist?.campaignId,
```

Comment:
I see above (line 50) we define `deliveryId` same as here but it also looks like in the next line we exit this function when `deliveryId` is defined and a non-empty string does that mean here it would always be empty or undefined and is that okay/expected? Should this check be ran first?

Reply:
Maybe like we do below: https://github.com/customerio/cdp-analytics-js/blob/b4ff67a17cd1ac7527ce1e9c209f45c4a82e462c/packages/browser/src/plugins/in-app-plugin/index.ts#L93 ?

Reply:
Prior to this change we would return the same event with deliveryId as undefined, this would only occur if the message is a test message or an anonymous message (broadcast).

But now that you raise this point i'm curious to know why we don't fire the `in-app:message-action` event when a message is anonymous. @clabland @karngyan any ideas?

Reply:
Hm. Feels like oversight on my part? `in-app:message-action` would need to learn about content events though (with their type and ids, the data  on line 113 below https://github.com/customerio/cdp-analytics-js/pull/40/files#diff-3021ca68b2a236327198b17a073486b5f4dd4ee0e2c085d6b9ce2fc3bcbca7ceR113